### ISSUE TITLE
amp-script: Mutating hidden attribute raises error in sanitizer

### ISSUE THREAD

**Issue by westonruter** [ampproject/amphtml#23156]
Given this JS used for `amp-script`:

```js
const toggleNavMenu = document.getElementById( 'toggleNavMenu' );
const navMenu = document.getElementById( 'navMenu' );

toggleNavMenu.addEventListener( 'click', () => {
    navMenu.hidden = ! navMenu.hidden;
} );
```

An error is raised when attempting to set `navMenu.hidden`.

![image](https://user-images.githubusercontent.com/134745/60536976-4829e280-9cbc-11e9-9236-e1ae2532a7c9.png)

In [Slack](https://amphtml.slack.com/archives/C99NYV631/p1562090536003200), @kristoferbaxter identified the problem was happening in the sanitizer:

```js
b.sanitizer.mutateAttribute(HTMLNavElement, 'hidden', null)
```

Mutating the `hidden` attribute is valid AMP, as it can be done with `amp-bind` via `[hidden]` or via an AMP action via `toggleVisibility`. It should be doable in `amp-script` as well.

**Comment by kristoferbaxter** [comment]
Looks like the sanitizer doesn't like the replacement of a value with `null`. Perhaps the sanitization request isn't correctly plumbed from the `AttributeExecutor` to the AMP Sanitizer?

![Screen Shot 2019-07-02 at 11 31 59 AM](https://user-images.githubusercontent.com/61764/60537342-2da43900-9cbd-11e9-9a42-09a9083381b6.png)

**Comment by choumx** [comment]
Dupe of #22371. Will be fixed by #23245 and https://github.com/ampproject/worker-dom/pull/569.
